ipu-speed-advantage/benchmarking_gnn_on_ipus_review.ipynb

```diff
@@ -0,0 +1,867 @@
x2NrG9jS1MepVTk88PGFVOX5qO0K8n//2Em620FFjml0WG1NSpPppzYrayASp6VvqFDc2NDLqpquCTvXX7mghK+dM41ZRek4ZJGdLf38cVUtT6xvOKQWc6LI85sRftV5PmYXpzGzMJ1p+X7Ksr0jYijjmukOLVssH9OzQkISRf6zt4PfrzjI37e2TDjfPRnnzSzglx+Zy5Q8Pwe7Anz0gbVsaOg99A2HwSWLfPeimdx4aqWZSKHq/HVzEwIkmtK2dhmG9lAAg9E4LX1h9nUE2NrUx6qablbXdBGKHZ3PXY7PwY8uncMV80tIcyu0D0S4d8VBHlhZS1PvUGRkQZqLr583jZtPq8Qw4PntLVz/p/VIgpCSmZuSzJDlSTFIHV4AN/SE2N8ZIMvj4LyZBeT4nLQPRPjKs1t5dG0DWV4HnzqpnM8vqaIs2/uOzSOTs2t1TScWixGPx838estLA96fTWwbH8S1/n1b4DpcQ5q5vXv38s9//pOXX36Z2tpaHn/8cebPn5+g+b4bG7+2gQgX3LeW2p5wYsoli3D21Dwe+8T8RIfUDqhPLMCanii2x1uAh1uFF339JbqHFR52X+0zJ5fz26vmpVKg9fE3vLaOz9b72hmDuq6DAV98djvPbG3B7ZSSRtQCqqbjkSVeuOUU9nUMUtMVNCfAVoh6XXfokDEHNpbPKeL3Hz2O/DQnr+3u4PI/rJzQyXFmYRrfWz6Ls6bl4Xcp/GN7C598ZD2hqDqCxi0KJLqwFTm+lI7s5BwvPdE4/2nsQba62oalfYrqOm4E2jtDbGk0T94TyZkTBfjoolJ+c9X8RIf20bX1PLy6LmWC+uDnT8TnUhJmUTY8Toln1zTy5Iq6Me/DIYtU5HkozfHikiTmFqazpDqXylwfJRnulO45WFpZy8ysNRRkf38fMPT+sZ+xyZ50MpzOEXTr5M7/4SBm6Y5TNd56wpEx2ZlR1QzOm1lAQZoLBIjGdf68qZF5kzPo1CIIgGB1WnTD7CovysnCaUUB9YbiNPSG2Ns2yPaWfjY09LC+rveINg+j4ZSqHD63pJLL55cgALqmTiji53DxQe7gTgSXXXE5v/3Nb1m/fj1pfj+nn346X/3aV/nxj94/i94xjI//doErylKCpdDZ2ckrr7zCP//5TzZu3Mgtt9zCrbfemmgAvZdF7fA11TCMRLE9nqlMyrRXTJ38jhYLYhig6gaSpam10RuKUdsVTKzLtdbaXNMVoLE3fMQGV1W5Xr530SzOm1lAhsdBKKbyl01NPLa2njf2doxoPiuSwJfPnsq3l83AMOBHr+zhO8P8ILK8ZtE7eRgrqzrPR/Ewp3+bvdQbitHYG+ZAR8A0H6rvIRZX+b+LZrFkSh4+p4yq66yt7eHf+zpo7Y9YcTgSPqeMzyHjtii0LisSx5YduRQzHcAhmxpmSRhdenQka6E94U7GkAGW6fPQ2GtNFO1YwsbeQzr9S6LAp0+azN2XzCLDrbDqYDdXPrBm3KxnG2dOzeN7y2exoDQThyyyqaGX3791kKc3No7Q6DplM+4uefo72UplKM/2kjPMv8JuJncHzUJxX3uA7S39rB8lbnE0XDK3iP+7cCbTC9OQRIHXdrfzuzcP8o8dreO+h8uyPHx72QyuPbGcuKbz1MZGbnx845g+KuluZUQqw3gFsGbp36OqZr03RDTVZFMdyTpsr+mKY1h2rWEkmtvJTWxN04fymT4A+CCs9e/LAjdxn5rK9ddfz5YtWwDweDyceeaZfOMb3yAzMxNIDUu2O6h2IWj/7kjRFYjyh9UNvHWgB5cicvncQq4+rgin1bmVlaGusl1sH+kCfNPT2/jzZjMGKUGLFsApifztphM5Y2r+qB3hFP2vNSmWRAlJNhdg+7iGL8DL7lvHqtoe0j0OnIppLBRTdQZCJi2q96cXAQI6AoospVCeTBt4Mxt3d/sgrf0RGnpC1HQFaO4LpyyIXqfEt5fN5H/OnELHYJhp33ll1GgHn0Pku8tn89FFpeT5h8wrHllTz0s7WiYc+zIc6W6Fm86ppijbbbpqW7RVtyRyXkUuBb6h7uVQp1nl+y/t4on1DbgVkahq4FYkMr0OeoNRphak8dxNJ/HW/k62t/TjkMWELsXjkPAoMkKGDIKACCSvlQZQ4HRQ5nGNSi/b2tzPN17YwWA0nihQHbLI7adXc8mc4nEfq6rpbO3pYiAWM+lH9n0aBjqQ7/YwIysLgM7BCPutjcT+jkBioplMNxpv2hlVtTGn7sk4blIGP7lsDidV5uBSJLY19fFbS1s7GFH5+bVzmFrkxyWb0USabhCIqsiSyMHmAIF+zaRb5ZoNi7ykbvPwTYQ92d7RYrolbxjFLTkZPqfMx08o5bbTq5lemEZc05EEkymSfB4Zok0e+eY6uYOraVqChqwoCg6Hgvg+piFPFDt37eQb3/gmf3vuOUKhED/40Q9RZIWFCxZwwQUXvGv3e6zAPbr4b6/1hmEgiCI//8XPefjhh9F1HUEQOP744/nyl7/MnDlzEtdNTjoY3vB9N2DHEymKkihq1bhK/Chq8CcqdbIfo6ab5/NksyVNN2jpD7O/I8DBzkCi8LWL4OGmhhkemTvPn8FHF5UmPDZe3d3Gn9bU8/y2FkKxQz+2ihwv916zgLOn59PUG+LSe1dNaMooClCU4U4UvxVJTsmVOT7y01yJx2R/TU5lGA92HE5E1Swt6lDzNRhTE1GCE51wD5+MOySB25ZWc+6MAvxOmUhc56kNDTy4uo51dT2UZLhH0J/ttSulUNTNSB07672+J8TejkG2NpoGWLvbhmRBfpfM/547jS+fNRWApzY08OlH14/YE2V4ZO5aPpurF04iy+tkIBLn4dV1/HFlLdua+w/53I0FtyIlHIUnJxWLU/J8lGWPFbeo0RmI0NwXob47SH1PkOkFaVwwq5AMj4OuQJT7VtTwwMqaCQ0WkjE13893L5rJlQsmEY6pPLCy9pDmlaPBjiacXuDna+dM47hSs6ZQ
```

Comment:
Remove because it is not appropriate for this section. 

```suggestion
```

Reply:
I'm not sure why the image is being highlighted for deletion. That is not intended :).